oeis.py

```diff
@@ -169,6 +151,27 @@ def phi(n):
     return [phi(x) for x in range(start, start + limit)]
 
 
+def A000040(start=0, end=999, plot=False):
```

Comment:
Missing: `@oeis` decorator to register it to the list of sequences.
Missing: A docstring indicating which sequence it is (see "oeis.py --list).